websites/0-9/0gomovies/presence.ts

```diff
@@ -0,0 +1,70 @@
+const presence = new Presence({
+		clientId: "1016086539861753937",
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let iFrameData: {
+	currTime: number;
+	duration: number;
+	paused: boolean;
+} = null;
+
+presence.on(
+	"iFrameData",
+	(data: { currTime: number; duration: number; paused: boolean }) => {
+		iFrameData = data;
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			startTimestamp: browsingTimestamp,
+			largeImageKey: "https://i.imgur.com/qldgMDR.png",
+		},
+		{ pathname, href } = document.location;
+	switch (pathname.split("/")[1]) {
```

Comment:
I'm suggesting you to store the `pathname.split("/")` since you use it multiple times.